reconcile/openshift_groups.py

```diff
@@ -111,6 +116,13 @@ def fetch_desired_state(oc_map):
     desired_state = []
 
     for r in roles:
+        exp_date = datetime.strptime(r['expirationDate'], '%Y-%m-%d').date()
+        if exp_date < date.today():
+            continue
+        elif (exp_date - date.today()).days > EXPIRATION_MAX:
+            raise RunnerException(
```

Comment:
Is this a `RunnerException` or just a good old `ValueError`?

Reply:
Yup, went ahead and updated that for you.